tutorial/README.md

```diff
@@ -295,8 +295,7 @@ docker-compose -f docker-compose-oracle.yaml down
 
 ### XStreams
 
-The connector can be used with the Oracle XStreams API but the `database.connection.adapter` must be specified with `xstream` to do so.
```

Comment:
Pointing out the option name wasn't neccessarily bad? I think I'll re-add that and merge.